gui/qt/main_window.py

```diff
@@ -3883,6 +3887,17 @@ def external_plugins_dialog(self):
         d.exec_()
         self.externalpluginsdialog = None # allow python to GC
 
+    def hardware_wallet_support(self):
+        if self.hardwarewalletdialog:
+            # NB: reentrance here is possible due to the way the window menus work on MacOS.. so guard against it
+            self.hardwarewalletdialog.raise_()
+            return
+        from .udev_installer import InstallHardwareWalletSupportDialog
+        d = InstallHardwareWalletSupportDialog(self, self.gui_object.plugins)
+        self.hardwarewalletdialog = d
+        d.exec()
```

Comment:
Minor nit: Does it get GC'd if you do this? I found I need to also setParent(None) for it to actually go away otherwise it may stick around.. ? (Not 100% sure about that -- I just remember sometimes QObjects with parent don't get GC'd always).